addon/components/rdf-input-fields/checkbox.js

```diff
@@ -9,7 +9,9 @@ export default class RdfInputFieldsCheckboxComponent extends SimpleInputFieldCom
     const matches = triplesForPath(this.storeOptions);
     if (matches.values.length > 0) {
       this.nodeValue = matches.values[0];
-      this.value = matches.values[0].value === '1'; // There is a bug in conversion from rdflib
+      // There is a bug in conversion from rdflib
+      this.value =
+        matches.values[0].value === '1' || matches.values[0].value === 'true';
```

Comment:
Just tested it quickly and the following seems to work as expected.

```suggestion
      this.value = Literal.toJS(this.nodeValue);
```

And then importing the util:

```js
import { Literal } from 'rdflib';
```